content/blog/quarkus-and-testcontainers.asciidoc

```diff
@@ -0,0 +1,142 @@
+---
+title: "Quarkus and Testcontainers"
+date: 2020-11-28T17:45:00+01:00
+draft: false
+---
+:source-highlighter: rouge
+:rouge-style: base16.dark
+:icons: font
+:imagesdir: /images
+ifdef::env-github[]
+:imagesdir: ../../static/images
+endif::[]
+
+The https://www.testcontainers.org/[Testcontainers] project is invaluable for spinning up containerized resources during your (JUnit) tests,
+e.g. databases or Kafka clusters.
+
+The project provides the https://www.testcontainers.org/quickstart/junit_5_quickstart/[`@Testcontainers`] JUnit extension for controlling the lifecycle of containers used by a test.
```

Comment:
this sounds like `@Testcontainers` and JUnit 5 is the only supported option - consider rephrasing to avoid confusing the readers :)

Reply:
Ok, will clarify that this is for JUnit 5 specifically.